libmamba/src/core/prefix_data.cpp

```diff
@@ -227,7 +227,6 @@ namespace mamba
             { "PYTHONIOENCODING", "utf-8" },
             { "NO_COLOR", "1" },
             { "PIP_NO_COLOR", "1" },
-            { "PIP_NO_PYTHON_VERSION_WARNING", "1" },
```

Comment:
`PIP_NO_PYTHON_VERSION_WARNING` is deprecated and is a no-op now: https://github.com/pypa/pip/issues/13154.